tensorflow/compiler/tf2xla/g3doc/generate_ops_coverage.py

```diff
@@ -0,0 +1,64 @@
+from datetime import date
+from google.protobuf import text_format
+from pytablewriter import MarkdownTableWriter # https://pypi.org/project/pytablewriter/
+import support_pb2 as Support
+
+
+today = date.today()
+today = today.strftime("%Y-%m-%d")
+
+supported_ops = Support.OpSupports()
+
+
+def parse_supported_ops():
+  with open("./support.pbtxt", 'r') as f:
```

Comment:
Where does this file come from?

Reply:
It is maintained by @rick at https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2xla/support.pbtxt but we need to compile the protobuf in the build system for importing it in  python (line 4).

Reply:
@ricmk Can you join in this thread?

Reply:
Hi @bhack, the support.pbtxt information has not been fully verified and is initially intended for deriving the XLA compilation support per TF op (and is not likely suitable for general coverage tables at the moment).

Reply:
> has not been fully verified and is initially intended for deriving the XLA compilation support per TF op

Is it manually populated? Or are you going to automate it using the [XLA op registry](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2xla/xla_op_registry.h) and other registers?

Also if your work is still WIP I still think that it is better then nothing that we produce a table as the official tables in the same folder are outdated by 4 years and I think it is enough time for a project like TF. 

So having a WIP table It is is better then just let the user fail the function compilation op after op (as you know we have a fail fast policy in `jit_compile`).

Reply:
Hi bhack@, this protobuf design is not yet stable and is subject to changes (these fields in particular will be removed shortly), so we do not want to introduce any dependencies on its field at this time. Thank you for your understanding.

Reply:
>  so we do not want to introduce any dependencies on its field at this time

We could modify the script as you change the proto it is just something that it is going to produce a markdown page where now we have something that is not updated by more then 4 years. I suppose that we don't need to find the perfection of the definitive proto when we have something so outdated.

Also you have not replied to my second point:
> Is it manually populated? Or are you going to automate it using the [XLA op registry](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2xla/xla_op_registry.h) and other registers?

Reply:
The protobuf is currently populated by a combination of static analysis and manual verification. We are relocating the protobuf shortly and will resurface the compilation data with more explicit meanings for the fields. Cheers.